refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-client/src/main/api/api-docs.json

```diff
@@ -126,26 +126,26 @@
           "FileAPI"
         ],
         "description": "Creates a presigned URL to delete the file specified in the parameter from the S3 storage",
-        "operationId": "delete_1",
```

Comment:
Is a bit out of scope but instead of renaming `delete_1` and `delete` in the client, as the generator seams to switch here, it's probably a better idea to rename all methods to contain File and Folder like `delete` to `deleteFile` and `deleteFolder`. That should fix that.

Reply:
I complete agree with you. I will do that within this branch.

Reply:
names of deletion methods changed

https://github.com/it-at-m/refarch/pull/473/commits/15d326fe70314b36601a76f379c11a24ba26cc5d